Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,39 @@
-function setAlarm() {}
+function setAlarm() {
+  // Accessing and storing input and display elements from html file
+  let timeDisplay = document.getElementById("timeRemaining");
+  let inputField = document.getElementById("alarmSet");
+
+  // Converting input value to a number
+  let seconds = parseInt(inputField.value);
+
+  // Checking if input is valid (not empty, not negative, and a number)
+  if (isNaN(seconds) || seconds <= 0) {
+    alert("Please enter a valid number of seconds.");
+    return;
+  }
+
+  function updateTimer() {
```

Comment:
Just a comment and no need to make a change here: What was your thought process behind defining updateTimer within the setAlarm function? In your code, this is a nice example of encapsulation (time update is tightly coupled to setting the alarm & probably won't be used in other places).